python-frontend/src/test/java/org/sonar/python/semantic/ProjectLevelSymbolTableTest.java

```diff
@@ -526,9 +526,9 @@ void test_imported_modules() {
     assertThat(projectLevelSymbolTable.importsByModule()).containsOnly(
       Map.entry("mod", Set.of("A")),
       Map.entry("mod2", Set.of("A", "B")),
-      Map.entry("mod3", Set.of("C.D.foo", "C.D.bar")),
+      Map.entry("mod3", Set.of("C.D")),
       Map.entry("mod4", Set.of("E")),
-      Map.entry("mod5", Collections.emptySet())
+      Map.entry("mod5", Set.of("F"))
```

Comment:
Note: If a relative import appears to go above the root of the project, its resolution will default to the project root and not go higher.